### PR TITLE
Vet clinic database: database performance audit

### PR DESCRIPTION
**Pull request by Yerimah**
In this branch we worked on the following:
- Added a new column
- Inserted data into the new column
- Used EXPLAIN ANALYZE on the queries to check the execution time
- Created Index to reduce the execution time
- Took screen-shots for the execution time before and after
- SELECT COUNT(*) FROM visits where animal_id = 4;
**Before Indexing**
animalbeforeshot
**After Indexing**
animalAfterShot

-SELECT * FROM visits where vet_id = 2;
**Before Indexing**
vetBeforeShot
**After Indexing**
vetAfterShot

- SELECT * FROM owners where email = 'owner_18327@mail.com';
**Before Indexing**
ownerBeforeShot
**After Indexing**
ownerAfterShot

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -67,4 +67,15 @@ CREATE TABLE visits (
...
\ No newline at end of file
+);
+
+-- Run the following query to add an extra column to the owners table:
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+--indexing 
+CREATE INDEX visits_animal_id_idx ON visits(animal_id);
+
+CREATE INDEX visits_vets_id_idx ON visits(vets_id);
```
Comment (SajjadAhmad14) [inline schema.sql:78]: For query : `SELECT * FROM visits where vet_id = 2`:
The screenshot with the title `BEFORE` shows an execution time of far less than 1000ms which should be around 1000ms.
Please try running the populate database query a few times more as described [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms.)

File: schema.sql [hunk 2]
```diff
@@ -67,4 +67,15 @@ CREATE TABLE visits (
...
+
+CREATE INDEX visits_owners_id_idx ON owners(email)
```
Comment (SajjadAhmad14) [inline schema.sql:80]: For query : `SELECT * FROM owners where email = owner_18327@mail.com](mailto:owner_18327@mail.com)`:
The screenshot with the title `BEFORE` shows an execution time of far less than 1000ms which should be around 1000ms.
Please try running the populate database query a few times more as described [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms.)

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi team, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾🎉✔**
Hello @Yerimah, @BertrandConxy, and @farouqdaif1 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**